static/app/types/core.tsx

```diff
@@ -78,8 +81,11 @@ export enum DataCategory {
   REPLAYS = 'replays',
 }
 
-// https://github.com/getsentry/relay/blob/master/relay-common/src/constants.rs
-// Should be used in conjuction with DATA_CATEGORY_INFO rather than manipulating the string
+/**
+ * https://github.com/getsentry/relay/blob/master/relay-common/src/constants.rs
+ * Matches the backend singular backend enum directly.
+ * For display variations, refer to `DATA_CATEGORY_INFO` rather than manipulating these strings
+ */
```

Comment:
Can we rename this to `DataCategoryRelay`? Or something signifying the source?
Maybe `DataCategoryKeys`?

`exact` isn't very clear to me about what it's being exact about 🤔

Reply:
That's a good call out, I have some gs code that references it unfortunately, but I can make that change here, but I'll keep it in mind when doing the wrap up PRs later.